### PR TITLE
Rewrite the reuse section

### PR DESCRIPTION
**Pull request by abelsiqueira**
Regarding #30. See discussion there.

### CODE REVIEW

File: reuse.tex [hunk 1]
```diff
@@ -1,11 +1,26 @@
...
+uses one of our own as a starting point.
 
-For additional support related to perprof-py, visit the project page on Github~\cite{url:perprof-py}. 
\ No newline at end of file
+Our parser opens the input files and creates the information for {\tt Pdata}.
+Replacing this parser (for use with out backends) would not be so easy
+because the user would have to create the correct output format.  However,
+extending it with additional options would be simple enough.
+
+Our binary {\tt perprof-py} essentially collects the options from the command
```
Comment (rgaiacs) [inline reuse.tex]: It's not a binary but a wrapper.

~~~
$ cat $(which perprof) 
#!/usr/bin/python3
# EASY-INSTALL-ENTRY-SCRIPT: 'perprof==1.0.0','console_scripts','perprof'
__requires__ = 'perprof==1.0.0'
import sys
from pkg_resources import load_entry_point

if __name__ == '__main__':
    sys.exit(
        load_entry_point('perprof==1.0.0', 'console_scripts', 'perprof')()
    )
~~~

Please use

> Our entry point, {\tt perprof-py}, essentially collects ...

File: reuse.tex [hunk 2]
```diff
@@ -1,11 +1,26 @@
...
+line and calls the specific backend profiler. This can be completely bypassed
+by calling the backend directly. This allows the user to create a performance
+profile from another python application. In particular, one possibility is the
+creation of a graphical user interface (GUI). The modularity of the package
```
Comment (rgaiacs) [inline reuse.tex]: "creation of a graphical user interface (GUI)." -> "creation of a graphical user interface (GUI) or a web server application".

**Comment by abelsiqueira** [comment]
Need to know if re-implementing that thing would break something and make sense.

**Comment by lrsantos11** [comment]
Ok. I agree. Let's merge a continue.

**Pull request merged**